.gitignore

```diff
@@ -1,5 +1,6 @@
 *.zip
 PowerShellEditorServices/
+Snippets/
```

Comment:
can we ship the snippets directly? :)

Reply:
I like the idea of piggy-backing off of the work done for vscode-powershell, and when the package.json is updated to have the snippets in there, it should ship them with the npm...

That being said, I think we can contribute multiple snippet files, so perhaps we can include a snippets file of our own?

Reply:
it makes sense to allow for multiple entries, or even custom entries in user-defined directories.
I'll poke around to see if there's a way to programmatically add snippets without populating package.json ahead of time.

Reply:
@yatli user-defined snippets is probably already a thing provided by coc.

I agree with @corbob that grabbing the vscode-powershell snippets is a good start here. If we want to add more on top of that, we can for sure in one of the other mechanisms

Reply:
I think we can also create quick ps module that would be adding snippets to PowerShell.json, so users can either update it manually or add their custom snippets. Like New-VimSnippet/Remove-VimSnippet/Import-VimSnippet (similar to ISE)

Reply:
I know vscode has a concept of user-defined snippets, I wouldn't be surprised if coc.nvim has it as well.

That'd be the best place a `New-VimSnippet` could target by default.